docs/user/python.md

```diff
@@ -74,6 +74,17 @@ stadium
 2
 ```
 
+User can pass the predefined object class id during dict() method call.
```

Comment:
it would be good to expand this and mention passing the oclass is optional; can be RP or EC or S; and has to satisfy the rf property on the container. Also one could create the container with RF setting indicating that the oclass used automatically would use replication.
- in your example it might encourage using EC for dictionaries, which we should not do ;-); so please use RP instead.

Reply:
I will extend the comments. Example has OC_RP_2G1 not EC, so it should be ok.